codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/endpoints/DefaultEndpointProviderTestGenerator.kt

```diff
@@ -61,6 +61,43 @@ class DefaultEndpointProviderTestGenerator(
     fun render() {
         writer.addImport("*", namespace = "kotlin.test")
         writer.withBlock("public class #L {", "}", CLASS_NAME) {
+            writer.write("")
+            writer.withBlock(
+                "private fun expectEqualEndpoints(expected: #T, actual: #T) {",
+                "}",
+                RuntimeTypes.SmithyClient.Endpoints.Endpoint,
+                RuntimeTypes.SmithyClient.Endpoints.Endpoint,
```

Comment:
simplification: You can remove the duplicated `RuntimeTypes.SmithyClient.Endpoints.Endpoint` by replacing `#T` with `#1T`

Same for all other lines where you duplicate the `#T`